chunky/src/java/se/llbit/chunky/ui/render/AdvancedTab.java

```diff
@@ -146,25 +149,39 @@ public PictureExportFormat fromString(String string) {
       renderControls.showPopup("This change takes effect after restarting Chunky.", renderThreads);
     });
 
-    ArrayList<String> implNames = new ArrayList<>();
+    ArrayList<String> octreeNames = new ArrayList<>();
     StringBuilder tooltipTextBuilder = new StringBuilder();
     for(Map.Entry<String, Octree.ImplementationFactory> entry : Octree.getEntries()) {
-      implNames.add(entry.getKey());
+      octreeNames.add(entry.getKey());
       tooltipTextBuilder.append(entry.getKey());
       tooltipTextBuilder.append(": ");
       tooltipTextBuilder.append(entry.getValue().getDescription());
       tooltipTextBuilder.append('\n');
     }
     tooltipTextBuilder.append("Requires reloading chunks to take effect.");
-    octreeImplementation.getItems().addAll(implNames.toArray(new String[implNames.size()]));
+    octreeImplementation.getItems().addAll(octreeNames);
     octreeImplementation.getSelectionModel().selectedItemProperty()
             .addListener((observable, oldvalue, newvalue) -> {
               scene.setOctreeImplementation(newvalue);
               PersistentSettings.setOctreeImplementation(newvalue);
             });
-    octreeImplementation.setTooltip(new Tooltip(
-            tooltipTextBuilder.toString()
-    ));
+    octreeImplementation.setTooltip(new Tooltip(tooltipTextBuilder.toString()));
+
+    ArrayList<String> bvhNames = new ArrayList<>();
+    StringBuilder bvhMethodBuilder = new StringBuilder();
+    for (String entry : BVH.Factory.getImplementationStrings()) {
+      bvhNames.add(entry);
+      bvhMethodBuilder.append(entry);
+      bvhMethodBuilder.append(": ");
+      bvhMethodBuilder.append(BVH.Factory.getImplementation(entry).getDescription());
+      bvhMethodBuilder.append('\n');
+    }
+    bvhMethodBuilder.append("Requires reloading chunks to take effect.");
+    bvhMethod.getItems().addAll(bvhNames);
+    bvhMethod.getSelectionModel().select(PersistentSettings.getBvhMethod());
+    bvhMethod.getSelectionModel().selectedItemProperty()
+            .addListener(((observable, oldValue, newValue) -> PersistentSettings.setBvhMethod(newValue)));
```

Comment:
Maybe ask the user to reload the chunks after changing this (see the octree code above for how to do that).
Also (just like the octree), should this also be a property of the scene?